pom.xml

```diff
@@ -93,6 +93,12 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
Didn't we agreed that client won't be dependent on the actual auth library? It's up to user to add a dependency if he needs, and provide an implementation that we offer to client instance?